src/main/webapp/ts/login.ts

```diff
@@ -0,0 +1,37 @@
+
+type authInfo = {
+    loginUrl: string;
+    logoutUrl: string;
+};
+//function that loads the home page for logging in and logging out
+async function loadHome() {
+    const logging = document.getElementById('log-status-section');
+    const link = document.getElementById('login-link');
+
+    const logStatus = await getLogStatus();
+
+    //set up function to set login/logout link based on which string is non empty
+    if (link && logging) {
+
+        if (logStatus.loginUrl === "") {
+            link.setAttribute('href', logStatus.logoutUrl);
+            console.log(logStatus.logoutUrl);
+            link.innerHTML = 'Logout';
+        } else {
+            link.setAttribute('href', logStatus.loginUrl);
+            link.innerHTML = 'Login';
+        }
+        logging.style.display = 'block';
```

Comment:
Any reason you need to set this in the script, rather than as a property of the css?
Also, 'log-status-section' has 'nav-item' as its class, but I can't seem to find it in the codebase (am I missing it?).